src/example.env

```diff
@@ -1,6 +1,34 @@
 # Backend API services, shown for DEV
-NEXT_PUBLIC_APP_BACKEND_URL = 'https://ingest-api.dev.hubmapconsortium.org'
-NEXT_PUBLIC_DATASET_URL = 'https://ingest-api.dev.hubmapconsortium.org/datasets/data-status'
-NEXT_PUBLIC_UPLOAD_URL = 'https://ingest-api.dev.hubmapconsortium.org/uploads/data-status'
-NEXT_PUBLIC_PORTAL_URL = 'https://portal.dev.hubmapconsortium.org'
-NEXT_PUBLIC_INGEST_URL = 'https://ingest.dev.hubmapconsortium.org'
\ No newline at end of file
+NEXT_PUBLIC_API_BASE = https://ingest-api.dev.hubmapconsortium.org
+NEXT_PUBLIC_DATASET_URL = https://ingest-api.dev.hubmapconsortium.org/datasets/data-status
+NEXT_PUBLIC_UPLOAD_URL = https://ingest-api.dev.hubmapconsortium.org/uploads/data-status
```

Comment:
I had noticed this earlier when @DerekFurstPitt created the data-ingest-board. @libpitt It's a good opportunity now that we expand the config variable for reuse purpose as below:

```
NEXT_PUBLIC_API_BASE = https://ingest-api.dev.hubmapconsortium.org
NEXT_PUBLIC_DATASET_URL = $NEXT_PUBLIC_API_BASE/datasets/data-status
NEXT_PUBLIC_UPLOAD_URL = $NEXT_PUBLIC_API_BASE/uploads/data-status
```